tests/Kernel/fixtures/rendering.yml

```diff
@@ -964,3 +964,32 @@
       'div.ecl-dialog[aria-hidden="false"] h3#ecl-dialog-title': "Dialog title"
       'div.ecl-dialog[aria-hidden="false"] p#ecl-dialog-description': "Dialog description"
       'div.ecl-dialog[aria-hidden="false"] div.ecl-dialog__body': "<p>Dialog body</p>"
+- array:
+    '#type': pattern
+    '#id': timeline
+    '#fields':
+      limit: 1
+      button_label: "Timeline button"
```

Comment:
We should test when we do not specify a button label. 
According to set default value here https://github.com/openeuropa/oe_theme/pull/268/files#diff-24773e644d42d1c7a2ec9a7643d80560R20